frontend/my-app/src/requests.ts

```diff
@@ -1,9 +1,10 @@
 import type { EquidistantRequest } from './requestObjects'
 import type { EquidistantResponse } from './responseTypes'
-export { makeRequest }
 
 async function makeRequest (request: EquidistantRequest): Promise<EquidistantResponse> {
   const response = await fetch(request.path, request)
   const json = await response.json()
   return json
 }
+
+export { makeRequest }
```

Comment:
since this is the only function in this file, I would probably just make it the default export
`export default async function makeRequest...`. I also think it might be a good idea to rename the file `makeRequest.ts` so it's clear this is the only thing in it... but if we're planning on adding more to this file then this can all be ignored lol

Reply:
Honestly not sure what else we might need. I think for now we can leave it and it should be a pretty easy change in the future should we not need anything else